setup.py

```diff
@@ -21,14 +21,22 @@
     author='Pusher',
     author_email='support@pusher.com',
     classifiers=[
-      'Development Status :: 3 - Alpha',
       'Intended Audience :: Developers',
+      'Programming Language :: Python',
+      'Programming Language :: Python :: 2',
+      'Programming Language :: Python :: 2.7',
       'Programming Language :: Python :: 3',
+      'Programming Language :: Python :: 3.4',
+      'Programming Language :: Python :: 3.5',
+      'Programming Language :: Python :: 3.6',
+      'Programming Language :: Python :: 3.7',
+      'Programming Language :: Python :: Implementation :: CPython',
     ],
     dependency_links=dependency_links,
     description='Pusher Push Notifications Python server SDK',
     include_package_data=True,
     install_requires=install_requires,
+    python_requires=">=2.7, !=3.0.*, !=3.1.*, !=3.2.*, !=3.3.*",
```

Comment:
I would also add an upper bound `<=3.7` for future safety. Also what's the reason for _not_ supporting `3.0.*`, `3.1.*`, `3.2.*`,  `3.3`?

Reply:
Fair enough on the upper bound (although why limit minors?)
We don't support 3.0 -> 3.3 because our dependencies don't (I guess because those versions weren't that mature)